lib/inspec/base_cli.rb

```diff
@@ -118,6 +118,8 @@ def self.target_options # rubocop:disable Metrics/MethodLength
         desc: "Disable SSL verification on select targets"
       option :target_id, type: :string,
         desc: "Provide a ID which will be included on reports"
+      option :winrm_shell_type, type: :string, default: "powershell",
+        desc: "Specify a shell type for winrm (eg. 'elevated' or 'powershell')"
```

Comment:
@catriona1 Quick description to fix (in the suggestion below). We're trying not to add more Latin in the docs and descriptions.
```suggestion
        desc: "Specify a shell type for winrm, such as "elevated" or "cmd""
```